### PR TITLE
chore: Changelog dependency cleanup

### PR DESCRIPTION
**Pull request by bitsandfoxes**
I think it should be like that for all future bumps. Let's see.

#skip-changelog

### CODE REVIEW

**Comment by tustanivsky** [comment]
@bitsandfoxes Should we clean up the release pages on GitHub as well?

**Comment by bitsandfoxes** [comment]
> @bitsandfoxes Should we clean up the release pages on GitHub as well?

Yes. If you're okay with the cleanup I'll make the changes on the releases.

**Comment by tustanivsky** [comment]
> Yes. If you're okay with the cleanup I'll make the changes on the releases.

That would be great, thank you

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,57 +18,18 @@
...
-  - [diff](https://github.com/getsentry/sentry-java/compare/6.10.0...6.11.0)
+- Bump Java SDK (Android) from v6.8.0 to v6.14.0 ([#165](https://github.com/getsentry/sentry-unreal/pull/165)), [#168](https://github.com/getsentry/sentry-unreal/pull/168), [#172](https://github.com/getsentry/sentry-unreal/pull/172), [#177](https://github.com/getsentry/sentry-unreal/pull/177), [#179](https://github.com/getsentry/sentry-unreal/pull/179), [#184](https://github.com/getsentry/sentry-unreal/pull/184), [#193](https://github.com/getsentry/sentry-unreal/pull/193), [#195](https://github.com/getsentry/sentry-unreal/pull/195), [#199](https://github.com/getsentry/sentry-unreal/pull/199), [#209](https://github.com/getsentry/sentry-unreal/pull/209), [#212](https://github.com/getsentry/sentry-unreal/pull/212)
+  - [changelog](https://github.com/getsentry/sentry-java/blob/main/CHANGELOG.md#6140)
+  - [diff](https://github.com/getsentry/sentry-java/compare/6.8.0...6.14.0)
+- Bump Cocoa SDK (iOS) from v7.31.2 to v7.31.3 ([#167](https://github.com/getsentry/sentry-unreal/pull/167)), [#176](https://github.com/getsentry/sentry-unreal/pull/176), [#216](https://github.com/getsentry/sentry-unreal/pull/216)
```
[the remark this answers is not in the document]
Comment (bitsandfoxes) [reply CHANGELOG.md]: Thanks! Good catch!

**Review by tustanivsky — APPROVED** [review]

**Pull request merged**